health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomPieChart.js

```
@@ -0,0 +1,315 @@
+    setActiveIndex(null);
+  };
+  const numberOfSlices = 10;
+  const chartData = useMemo(() => {
+    if (!response) return null;
+    if (setChartDenomination != null) {
+      setChartDenomination(response?.responseData?.data?.[0]?.headerSymbol);
+    }
+    const compareFn = (a, b) => b.value - a.value;
+    return drillDownId === "deathByCategoryDrilldownAge" || response?.responseData?.visualizationCode === "nssNumberOfDeathsByAge" // || drillDownId === "nssDeathByCategoryDrillDownAge")
+      ? response?.responseData?.data?.[0]?.plots.reduce((acc, plot, index) => {
+          acc = acc.concat(plot);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)